events/wc_dark_cult_events.txt

```diff
@@ -606,7 +606,7 @@ province_event = {
 
 	immediate = {
 		# Starts "The Voice from Beyond the Sea" event chain
-		province_event = { id = WCDAC.5029 days = 1825 }
+		province_event = { id = WCDAC.5029 months = 120 }
```

Comment:
Cho'gall sailed to Kalimdor around the end of the Second War, so I would change it to at least...
```suggestion
		province_event = { id = WCDAC.5029 years = 6 }
```

Reply:
if we do that the journey will be too long

Reply:
I don't mind less time you know.

Reply:
so its ok to free c'thun earlier?

Reply:
@sejtalk I wouldn't do it because the Tol'vir needs time to prepare for the Qiraji swarm.

Reply:
`I don't mind less time you know.`

Reply:
I'm speaking of sailing to Kalimdor, not freeing C'thun.

Reply:
well considering sailing takes into consideration time spend before freeing him its the same thing

Reply:
The same thing? They're parallel event chains. Both fired by WCFTW.100.

Reply:
Cho'gall freed C'Thun the moment he arrived in kalimdor, 22 years after opening of the dark portal

Reply:
> if we do that the journey itself will be too long

> Cho'gall freed C'Thun the moment he arrived in kalimdor, 22 years after opening of the dark portal

> so its ok to free c'thun earlier?

I don't understand your point. You say absolutely opposite points of view.

Reply:
> Cho'gall freed C'Thun the moment he arrived in kalimdor, 22 years after opening of the dark portal

I told how it works in the mod, why you say it to me?

Reply:
If you have journey trait you have a regent so the waiting time for event chain shouldnt be reduced as that would make regency way longer

Reply:
The event chains final event that frees cthun will be added with freeing old gods mechanic

Reply:
Decrease the regency time. I told it in the comment below.

Reply:
that would break the timeline

Reply:
`province_event = { id = WCDAC.5029 months = 120 }`
This breaks the timeline too. 6 years, not 10 years.

Reply:
ok im fixing timeline